### DISCUSSION TITLE
Provider-Backend-Service - Deprecated?

### DISCUSSION THREAD

**Discussion by ds-mwesener** [eclipse-tractusx/tractusx-edc#215] [General]
Dear community, in our project: https://github.com/eclipse-tractusx/traceability-foss
we are currently still using the image: ghcr.io/catenax-ng/catenax-at-home/provider-backend-service

As this image is in a deprecated repository https://github.com/catenax-ng/catenax-at-home

I would like to know if there is something new available or if it is worth to maintain that image in a new eclipse-tractusx repository if multiple teams using this service.

**Comment by florianrusch-zf** [comment]
Hey @ds-mwesener, quick answer: no

The long explanation for this: Catenax-at-home was just a demo to teach people how the EDC works. None of the components were meant to be used in any kind of production environment. If you need something like the backend-service, you need to create, maintain and support it yourself.

**Reply by ds-mwesener** [reply]
Hi @florianrusch-zf thanks for the quick answer. 
Basically we do not use this image in any production environment.
Rather we use it for our test environments. 

Do you have any kind of documentation about this service? I cannot find anything.
If this will not be part of eclipse-tractusx the teams which using this service needs some guideance of how to replace it with something else - for testing purpose.

**Reply by florianrusch-zf** [reply]
The service is **very very very** basic. If you take a look at the code, I think you should be able to maintain it on your own: https://github.com/catenax-ng/catenax-at-home/blob/main/api-wrapper/services/backend-data-service/src/main/java/net/catenax/edc/bds/DataController.java

The service was never intended to be a "product" and was never communicated as such. Accordingly, the following applied from the very beginning: Use at your own risk

**Reply by ds-mwesener** [reply]
Ah that is a good reference. Actually I have already tried to find the code. 

Thanks I will adress that in our project to get rid of the deprecated chart.

Kind regards.
